src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -251,7 +251,7 @@ def default_api_version(self):
             'subscription_diagnostic_settings': '2017-05-01-preview'
         }),
         ResourceType.MGMT_MSI: '2023-01-31',
-        ResourceType.MGMT_APPSERVICE: '2022-03-01',
+        ResourceType.MGMT_APPSERVICE: '2023-01-01',
```

Comment:
I would remove the changes in this file, the requirements files and the setup.py file since we will create a separate PR for them.

Reply:
Yeah! I accidently checked it in. I will take it out.